File: C#/RemoveDuplicatesFromSortedArray.cs
public class Solution26 {
    public int RemoveDuplicates(int[] nums) {
        int curr = 0;

        for (int i = 0; i < nums.Length; i++)
        {
            if(nums[i]!=nums[curr])
            {
                nums[curr+1] = nums[i];
                curr++;
            }
        }
        
        if(nums.Length==0)
        {
            return 0;
        }
        else
        {
            return curr+1;    
        }
        
    }
}

File: Golang/Remove Duplicates from Sorted Array.go
func removeDuplicates(nums []int) int {
    count := 1
    for i :=1; i < len(nums);i ++{
        if nums[i] > nums[count-1] {            
            nums[count] = nums[i]
            count = count + 1
        }
    }
    return count
}


func removeDuplicates(nums []int) int {
    j := 1
    for i:=1; i < len(nums); i++ {
        if nums[i] != nums[i-1] {
            nums[j] = nums[i]
            j++
        }
    }
    return j
}

func removeDuplicates(nums []int) int {
    
    insert := 1
    for i:=1;i < len(nums);i++ {
        if nums[i] != nums[i-1] {
            nums[insert] = nums[i]
            insert = insert + 1
        }
    }
    return insert
}